modules/ytp-assets-common/src/less/ckan/upstream_ckan/forms.less

```diff
@@ -596,6 +596,16 @@ select[data-module="autocomplete"] {
     margin-left: 10px;
 }
 
+@media (max-width: @screen-xs-max) {
+    .slug-preview {
+        margin-left: -40px;
```

Comment:
I'm not fan of negative margins (although it is valid css). I wonder if there is an other option to move the slug to left? Also using negative margin kinda breaks the "grouping" of the form groups here. Maybe the slug content just needs to be broken in couple lines for smaller screen sizes.

Reply:
I removed the negative margin and instead reduced the padding on small screens.